netbox/resource_netbox_tenant_group_test.go

```diff
@@ -0,0 +1,105 @@
+package netbox
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/fbreckle/go-netbox/netbox/client"
+	"github.com/fbreckle/go-netbox/netbox/client/tenancy"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func testAccNetboxTenantGroupTagDependencies(testName string) string {
```

Comment:
This function is unused?

Reply:
I recommend removing it since tags do not seem to be implemented for this resource